### PR TITLE
Remove unused strings from source

### PR DESCRIPTION
**Pull request by dkanada**

### CODE REVIEW

File: scripts/duplicates.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    strings = json.load(en)
+    for key, value in strings.items():
+        if value not in reverse:
+            reverse[value] = [key]
+        else:
+            reverse[value].append(key)
+
+for key, value in reverse.items():
+    if len(value) > 1:
+        duplicates[key] = reverse[key]
```
Comment (dmitrylyzo) [inline scripts/duplicates.py:24]: `value` is `reverse[key]`, no?

File: src/strings/en-us.json [hunk 2]
```diff
@@ -93,7 +91,6 @@
     "ButtonHelp": "Help",
     "ButtonHome": "Home",
     "ButtonInfo": "Info",
-    "ButtonLearnMore": "Learn more",
```
Comment (dmitrylyzo) [inline src/strings/en-us.json:96]: https://github.com/jellyfin/jellyfin-plugin-fanart/blob/8bb2c0ddd4e5caf9a3ac5c1c894e8158040300f0/Jellyfin.Plugin.Fanart/Configuration/configPage.html#L16

File: scripts/duplicates.py (hunk 1 above)
Comment (dkanada) [inline scripts/duplicates.py:24]:
```suggestion
        duplicates[key] = value
```

Comment (dkanada) [reply scripts/duplicates.py]: good catch

File: src/strings/en-us.json (hunk 2 above)
Comment (dkanada) [reply src/strings/en-us.json:96]: The current translation method is a complete hack, so we shouldn't encourage plugins to be using it right now in my opinion.

**Pull request merged**